quilt/tools/command.py

```diff
@@ -721,11 +737,94 @@ def install(package, hash=None, version=None, tag=None, force=False):
 
     pkgobj.save_contents()
 
+def _setup_env(env, files):
+    """ process data distribution. """
+    # TODO: build.yml is not saved in the package system, so re-load it here
+    environments = None
+    with open('build.yml') as fd:
+        buildfile = next(yaml.load_all(fd), None)
+        environments = buildfile.get('environments')
+    if env != 'default' and (environments is None or env not in environments):
+        raise CommandException(
+            "environment %s not found in environments: section of build.yml" % env)
+    if environments is None:
+        return files
+    if env == 'default' and 'default' not in environments:
+        return files
+    print('processing environment %s: checking data...' % (env))
+    environment = environments[env]
+    dataset = environment.get('dataset')
+    for key, val in files.items():
+        # TODO: debug mode, where we can see which files were skipped
+        if type(val) == pd.core.frame.DataFrame:
+            before_len = len(val)
+            res = exec_yaml_python(dataset, val, key, '('+key+')')
+            if res == False:
+                raise BuildException("Data check failed: %s on %s @ %s" % (
+                    check, rel_path, target))
+            print('%s: %s=>%s recs' % (key, before_len, len(qc.data)))
+            files[key] = qc.data
+    print('processing environment %s: slicing data...' % (env))
+    instance_data = environment.get('instance_data')
+    for key, val in files.items():
+        # TODO: debug mode, where we can see which files were skipped
+        if type(val) == pd.core.frame.DataFrame:
+            before_len = len(val)
+            val['.qchash'] = val.apply(lambda x: abs(hash(tuple(x))), axis = 1)
+            res = exec_yaml_python(dataset, val, key, '('+key+')')
+            if res == False:
+                raise BuildException("Data check failed: %s on %s @ %s" % (
+                    check, rel_path, target))
+            print('%s: %s=>%s recs' % (key, before_len, len(qc.data)))
+            files[key] = qc.data
+    return files
+
+def load_to_dict(pkg, force=False, env='default', **kwargs):
+    import importlib
+    install(pkg, force=True)
+    user, module = pkg.split('/')
+    module = importlib.import_module("quilt.data.{}.{}".format(user, module))
```

Comment:
Take a look at the code in `data.py` and use store directly, instead of importing the module.

Reply:
DONE.  nice!